### PR TITLE
added html dialog

### PR DESCRIPTION
**Pull request by samthor**
Added a markdown file for the HTML dialog element, supported via polyfill.

### CODE REVIEW

File: posts/dialog.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+feature: dialog
```
Comment (arthurvr) [inline posts/dialog.md]: I'd prefer to be [consistent with other tags here](https://raw.githubusercontent.com/h5bp/html5please/master/posts/details.md), and wrap the element name in html-like syntax.

File: posts/dialog.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+status: use
```
Comment (arthurvr) [inline posts/dialog.md]: `caution` maybe? Use doesn't seem appropriate to me (yet)

**Comment by arthurvr** [comment]
Thanks for the PR, @samthor I'll review it this evening.

File: posts/dialog.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
```
Comment (arthurvr) [inline posts/dialog.md]: I saw you're one of the guys behind `dialog-polyfill`. Awesome stuff! Could we maybe add a note about the browser support of the polyfill itself? What is its official browser support (of the polyfill)?

File: posts/dialog.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+The `<dialog>` element can be used to create popup dialogs or modal dialogs for web applications, and is part of (HTML5.1)(http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html#the-dialog-element).
+  
+In its modal mode, it blocks all access to content behind the front-most dialog.
+The majority of [Google's polyfill](https://github.com/GoogleChrome/dialog-polyfill) goes towards preventing the browser from gaining focus or accessing content behind the dialog. You should consider that the preventative measures taken by the polyfill are not exhaustive, as they are not implemented natively.
+
+Supported in Chrome 37+ and Opera 24+, and [tracked](https://bugzilla.mozilla.org/show_bug.cgi?id=840640) for Firefox.
```
Comment (arthurvr) [inline posts/dialog.md]: Might be worth it to add the current status for IE too.

File: posts/dialog.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+feature: dialog
+status: use
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
+
+The `<dialog>` element can be used to create popup dialogs or modal dialogs for web applications, and is part of (HTML5.1)(http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html#the-dialog-element).
```
Comment (arthurvr) [inline posts/dialog.md]: Super nitpick but there must be a space between `HTML` and `5.1`.

File: posts/dialog.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  
+In its modal mode, it blocks all access to content behind the front-most dialog.
```
Comment (arthurvr) [inline posts/dialog.md]: Shouldn't this sentence be in the previous paragraph? In that paragraph we give a quick intro to the `dialog` element, in this one we're talking about the polyfill.

File: posts/dialog.md (hunk 4 above)
Comment (samthor) [inline posts/dialog.md]: There's no status for IE or Safari AFAIK. Is it worth enumerating all browsers that don't have support? Each element on html5please comes 'for free' with a link to caniuse.com.

File: posts/dialog.md (hunk 6 above)
Comment (samthor) [inline posts/dialog.md]: Done

File: posts/dialog.md (hunk 5 above)
Comment (samthor) [inline posts/dialog.md]: Done

File: posts/dialog.md (hunk 2 above)
Comment (samthor) [inline posts/dialog.md]: Done

File: posts/dialog.md (hunk 1 above)
Comment (samthor) [inline posts/dialog.md]: Done

File: posts/dialog.md (hunk 3 above)
Comment (samthor) [inline posts/dialog.md]: I've only recently inherited the polyfill - it seems like the original authors didn't list browser compatibility. I'll have to do some testing to confirm, but I think it's more appropriate to list the supported browsers on the `dialog-polyfill` page (none of the other items on html5please seem to list polyfill requirements).
How about I offer to make sure the compatibility list on our side is up-to-date? :-)

**Comment by arthurvr** [comment]
This post LGTM now

**Comment by samthor** [comment]
Can we merge it? :)

**Comment by arthurvr** [comment]
If nobody disagrees, I will do it this evening.

File: posts/dialog.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+feature: <dialog>
+status: caution
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
+
+The `<dialog>` element can be used to create popup dialogs or modal dialogs for web applications, and is part of (HTML 5.1)(http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html#the-dialog-element).
```
Comment (arthurvr) [inline posts/dialog.md]: > part of

I think you indented to have a link in there, isn't it?

Comment (samthor) [inline posts/dialog.md]: Whoops, fixed.

**Comment by samthor** [comment]
Ping?

File: posts/dialog.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+feature: <dialog>
+status: caution
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
+
+The `<dialog>` element can be used to create popup dialogs or modal dialogs for web applications, and is part of [HTML 5.1)](http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html#the-dialog-element).
```
Comment (arthurvr) [inline posts/dialog.md]: Seems like the link isn't totally right yet. There's still an extra bracket in inside the square ones.

Comment (arthurvr) [inline posts/dialog.md]: fixed it while mergin

**Comment by arthurvr** [comment]
@samthor Merged. Once again, thanks for the PR and sorry it took that long.

**Comment by samthor** [comment]
Thanks!

**Pull request closed without merging**